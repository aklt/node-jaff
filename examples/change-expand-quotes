#begin coffee
value = "A nice text"
#end

Here is a default quoted value: {value}

Here are the default quotes: {__expand}

#begin coffee

__quotes '<', '>'

#end

Old quotes don't work anymore: {value}

The new ones do: <value>

TODO: __expand does not hold the right quotes:  <__expand>
