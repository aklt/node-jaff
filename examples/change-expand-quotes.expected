
Here is a default quoted value: A nice text

Here are the default quotes: {
}


Old quotes don't work anymore: {value}

The new ones do: A nice text

TODO: __expand does not hold the right quotes:  {
}
